mir_eval/util.py

```diff
@@ -686,6 +686,40 @@ def validate_events(events, max_time=30000.):
         raise ValueError('Events should be in increasing order.')
 
 
+def validate_frequencies(frequencies, max_freq=5000., min_freq=20.):
+    """Checks that a 1-d frequency ndarray is well-formed, and raises
+    errors if not. Allows negative frequency values.
```

Comment:
How come it allows negative frequency values?  Wouldn't they throw off your eval?